frontends/web/src/routes/buy/iframe.module.css

```diff
@@ -1,21 +1,17 @@
 .container {
     display: flex;
-    flex-basis: 100%;
-    flex-grow: 0;
-    flex-shrink: 1;
-    flex-direction: column;
-    overflow: hidden;
-    position: relative;
+    width: 100%;
+    height: 100vh;
```

Comment:
please check that the fix here doens't break, see this commit https://github.com/digitalbitbox/bitbox-wallet-app/commit/a11b6d60490e54e3dd8f4ec98dcb5065505e89a1

Reply:
iframe scrolling seems to be okay.

https://user-images.githubusercontent.com/28676406/216982735-d59d8274-8750-46c7-992b-ce47e608189a.mp4